web-client/public/js/update-app.js

```diff
@@ -479,23 +477,25 @@ const expandLayoutSidebar = () => {
     $(LAYOUT_SIDEBAR_PANEL).addClass("in");
 };
 
-const toggleLayout = (on, off) => {
-    if (!$(on).prop("checked")) {
-        $(on).prop("checked", true);
-        $(off).prop("checked", false);
-        $(`${off} span`).removeClass("glyphicon-ok");
-        $(`${on} span`).addClass("glyphicon-ok");
-    }
-};
+// const toggleLayout = (on, off) => {
+//     if (!$(on).prop("checked")) {
+//         $(on).prop("checked", true);
+//         $(off).prop("checked", false);
+//         $(`${off} span`).removeClass("glyphicon-ok");
+//         $(`${on} span`).addClass("glyphicon-ok");
+//     }
+// };
```

Comment:
Since git will preserve this code anyway, it is safe to flat out remove it rather than commenting it out to keep things more readable